docs-v2/guides/webhooks/webhooks-from-apis/real-time-syncing-with-webhooks.mdx

```diff
@@ -45,15 +45,6 @@ hubspot:
 			  - contact.propertyChange
 ```
 
-Use `'*'` to list for any webhook subscription, e.g. 
-```yaml
-airtable:
-  syncs:
-    airtable-sync-name:
-	    webhook-subscriptions:
-			  - '*'
-```
```

Comment:
We don't want that to be documented?

Reply:
I would like to, but it's not true :) It's an Airtable-specific notation that I thought was Nango-specific.